polygeist/include/mlir/Dialect/Polygeist/Transforms/Passes.td

```diff
@@ -16,6 +16,40 @@ def AffineCFG : Pass<"affine-cfg"> {
   let constructor = "mlir::polygeist::replaceAffineCFGPass()";
 }
 
+def AffineReduction : Pass<"detect-reduction"> {
```

Comment:
suggestion: rename all `detect-reduction` to `affine-reduction` or `AffineReduction` to `DetectReduction`.

Reply:
Agree but also better to separate that in a different PR.